HowToAddANewService.md

```diff
@@ -0,0 +1,157 @@
+Tutorial on how to add a badge for a service
+============================================
+
+This tutorial should help you add a service to shields.io in form of a badge.
+You will need to learn to use JavaScript, git and Github. 
+
+(1) Reading
+-----------
+
+You should read the following:
+
+- [CONTRIBUTING.md](CONTRIBUTING.md)
+- [INSTALL.md](INSTALL.md)
```

Comment:
Most of INSTALL.md is irrelevant. Could you mention it below, for further reading?